solr/core/src/java/org/apache/solr/search/CollapsingQParserPlugin.java

```diff
@@ -276,7 +281,19 @@ public int getCost() {
     }
 
     public String toString(String s) {
-      return s;
+      return "!collapse field=" + this.collapseField +
```

Comment:
Maybe add curly braces for beginning and end

Reply:
Added them